src/Microsoft.DotNet.Interactive/KernelInvocationContextExtensions.cs

```diff
@@ -97,29 +97,4 @@ public static void DisplayStandardError(
                 command ?? context.Command,
                 formattedValues));
     }
-
-    public static void PublishValueProduced(
-        this KernelInvocationContext context,
-        RequestValue requestValue,
-        object value)
```

Comment:
This helper seems like it could still be useful to eliminate some boiler plate. Is this being removed to reduce public surface area?

Reply:
This helper code is much more verbose than necessary. I compressed it and inlined it. 

In general, helper methods for simple operations should not be part of the public API, especially considering:

* It's specific to one of dozens of possible events, so there's an inconsistency about why this event type and not others have such helpers.
* In IntelliSense, it distracts from the general-purpose `Publish` method.